app/views/councilTaxUpload.scala.html

```diff
@@ -43,11 +43,12 @@ <h1 class="govuk-label-wrapper">
     @components.error_summary(fileUploadForm.errors)
 
     @initiateResponse.map { ir =>
+        <input type="hidden" id="councilTaxUploadReportError" value=@(routes.CouncilTaxUploadController.onError(ir.reference)) />
```

Comment:
There was a slight difference with the original version:
https://github.com/hmrc/voa-bar-xml-frontend/pull/139/files?file-filters%5B%5D=.html#diff-1f4ca76d5f1fd583d360852f49be4d40L45

If this doesn't work - I ll remove the CSRF token as @Stephen-Goddard pointed out